python/ucxx/_lib_async/application_context.py

```diff
@@ -201,6 +205,13 @@ def create_listener(
             but prevents a process from terminating unexpectedly that may
             happen when disabled. If `False` endpoint endpoint error handling
             is disabled.
+        exchange_peer_info_timeout: float
+            Timeout for exchanging peer info. In some cases, exchanging peer
```

Comment:
```suggestion
            Timeout (in seconds) for exchanging peer info. In some cases, exchanging peer
```

Let's indicate what `10` means.